client/my-sites/site-settings/disconnect-site/pagination-flow.jsx

```diff
@@ -0,0 +1,44 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'gridicons';
+import Main from 'components/main';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+class PaginationFlow extends Component {
+	handleClick = () => {
```

Comment:
Do we want to use `redirectToGeneral` from the `redirectNonJetpackToGeneral` HoC here? Could be nice, because we wouldn't need to connect this component, and we could convert it to a stateless functional component.

Reply:
Ah yes, this is a great idea. It calls for an extension of that HoC's definition to redirect to a prop-passed url, because here we redirect to the next flow steps, not '/general'. Np, since that is the plan anyway 👍

Reply:
UPDATE: done https://github.com/Automattic/wp-calypso/pull/17555/